tests/schema-tests/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@sap-ai-sdk/schema-tests",
+  "version": "1.0.0",
+  "description": "Tests leveraging zod to ensure correct schema in the SAP AI SDK for JavaScript.",
+  "type": "module",
+  "private": true,
+  "types": "test",
+  "scripts": {
+    "test": "pnpm generate-schema && NODE_OPTIONS=--experimental-vm-modules jest",
+    "lint:fix": "prettier . --config ../../.prettierrc --ignore-path ../../.prettierignore -w --log-level error",
+    "generate-config:azure-openai": "node --loader ts-node/esm ../../scripts/generate-zod-config.ts '../../packages/foundation-models/src/azure-openai/client/inference/schema/*.ts' ./test/foundation-models/azure-openai/ts-to-zod",
+    "generate-config:orchestration": "node --loader ts-node/esm ../../scripts/generate-zod-config.ts '../../packages/orchestration/src/client/api/schema/*.ts' ./test/orchestration/ts-to-zod",
+    "generate-schema:azure-openai": "pnpm generate-config:azure-openai && pnpm ts-to-zod --all --skipValidation && rm ./ts-to-zod.config.cjs",
+    "generate-schema:orchestration": "pnpm generate-config:orchestration && pnpm ts-to-zod --all --skipValidation && rm ./ts-to-zod.config.cjs",
+    "generate-schema": "pnpm generate-schema:azure-openai && pnpm generate-schema:orchestration"
+  },
+  "tsd": {
+    "directory": "test"
```

Comment:
[req] Is not relevant here. Same for `"types": "test",` in ln. 7

Reply:
😄 Now you know from where I copied the package. Thanks and removed.